### ISSUE TITLE
Loading Groups while creating a new Message throws an Exception

### ISSUE THREAD

**Issue by ChMat** [claroline/MessageBundle#15]
Hi,

When a teacher creates a new message and tries to load the « Groups » tab, the exception below is raised:

```
php.CRITICAL: Fatal Error: Nesting level too deep - recursive dependency? {"type":1,"file":"/var/www/vendor/claroline/message-bundle/Claroline/MessageBundle/Controller/MessageController.php","line":611,"level":6143,"stack":[]} []
request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Error: Nesting level too deep - recursive dependency?" at /var/www/vendor/claroline/message-bundle/Claroline/MessageBundle/Controller/MessageController.php line 611 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Nesting level too deep - recursive dependency? at /var/www/vendor/claroline/message-bundle/Claroline/MessageBundle/Controller/MessageController.php:611)"} []
```

**Comment by ChMat** [comment]
Hi,

@ngodfraind I'm not yet used to pull requests, can you tell me if it will be merged as well on branch 5.0.*?

Thanks!

**Comment by ngodfraind** [comment]
You made the pr on the master. I merged it and changed the line myself for the 5.x since it was a very small change.

If you want it to be on the 5.x branch, you should target the 5.x-stable when making a pr.
